docs/src/main/sphinx/security/internal-communication.md

```diff
@@ -122,6 +123,12 @@ window functions, which require repartitioning), the performance impact can be
 considerable. The slowdown may vary from 10% to even 100%+, depending on the network
 traffic and the CPU utilization.
 
+:::{note}
+By default, internal communication with SSL/TLS enabled uses HTTP/2 for
+increased performance. You can turn off this feature with
```

Comment:
performance gains are not that important, I'd say scalability rather than a performance

Reply:
great .. done